app/Models/Course.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Course extends Model
+{
+    use HasFactory;
+
+
+    public $timestamps = true;
+
+    public function courses()
+    {
+        return $this->belongsTo('App\Category', 'category_id');
```

Comment:
return $this->belongsTo('App\Category', 'category_id');
=>
return $this->belongsTo(Category::class);